client/components/site-title/index.jsx

```diff
@@ -45,17 +45,19 @@ class SiteTitleControl extends React.Component {
 				<FormFieldset>
 					<FormLabel htmlFor="blogname">{ this.props.translate( 'Site Title' ) }</FormLabel>
 					<FormTextInput
```

Comment:
Both fields need an `ID` in order to be focused by clicking the label.

Reply:
Dropping `name`s in favor of `id`s per 3055e2f.